File: 00-warmup/fizzbuzz.py
def f(i):
  divX = i % x == 0
  divY = i % y == 0

  if divX and divY:
    return "FizzBuzz"
  if divX:
    return "Fizz"
  if divY:
    return "Buzz"
  return i

(x, y, n) = map(int, input().split(" "))
for i in range(1, n+1):
  print(f(i))

File: 00-warmup/fizzbuzz.cpp
#include <iostream>
typedef long long ll;
using namespace std;
int main(void) {
  ll x, y, n;
  cin >> x >> y >> n;
  ll num;
  for (int i = 1; i <= n; i++) {
    bool f = i % x == 0;
    bool b = i % y == 0;
    if (f && b) cout << "FizzBuzz\n";
    else if (f) cout << "Fizz\n";
    else if (b) cout << "Buzz\n";
    else cout << to_string(i)+"\n";
  }
  return 0;
}